.changeset/real-lies-remember.md

```diff
@@ -0,0 +1,15 @@
+---
+"@hopper-ui/icons": major
+---
+
+Breaking change: Updated the icons variables to be more consistent with the other components.
```

Comment:
I think since the component tokens were never exposed nor documented, and it's internal to our component developement, we should pass the update as a patch.